src/foam/graphics/TreeGraph.js

```diff
@@ -48,9 +48,8 @@
 
        // List for 'click' events to expand/collapse Nodes.
        this.canvas.on('click', function(e) {
-
          var x = e.layerX+this.nodeWidth/2, y = e.layerY;
-         var c = this.root.findFirstChildAt(x, y);
+         var c = this.root.findFirstChildAt(x/this.canvas.cview.scaleX, y/this.canvas.cview.scaleY);
```

Comment:
Hmm, I'm suprised that findFirstChildAt doesn't account for the scale, it's supposed to.

Reply:
Yeah had to add it otherwise it wouldn't work when zoomed in or zoomed out.